images/kubekins-e2e/variants.yaml

```diff
@@ -31,6 +31,12 @@ variants:
     K8S_RELEASE: stable
     BAZEL_VERSION: 3.4.1
     OLD_BAZEL_VERSION: 2.2.0
+  '1.29':
+    CONFIG: main
```

Comment:
This should be `1.29`, please see the follow-up PR on #30215: #30218. Otherwise, images are not going to be tagged properly.

Reply:
ack, fixed.